### PR TITLE
Add "Remove all accounts" testing steps

### PR DESCRIPTION
**Pull request by exigow**
Motivation: We need to describe this scenario explicitly because during the last release, this part was completely missed, and the issue was (fortunately) discovered by accident 1 hour before the release.

Also, I simplified the SSO testing because we had three headers for essentially the same thing.

## Test plan

N/A

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -51,15 +50,19 @@ Prerequisite: You have to **sign out** from all existing accounts.
...
 
-Onboarding through GitLab is similar to [Sign in with GitHub](#sign-in-with-github), and the authorization is also done
-through the browser. Expected behaviour is identical.
+1. Navigate to `Settings` > `Sourcegraph & Cody`.
+2. Remove all accounts and apply settings.
+
+#### Expected behaviour
+
+* Cody toolbar is automatically refreshed and the user is greeted with the `Welcome to Cody` panel.
+* Status bar widget has a `No account signed-in` status. Status bar is located in the bottom right corner of the IDE.
```
Comment (exigow) [inline TESTING.md:66]: Question: Should we expect anything more here?

File: TESTING.md [hunk 2]
```diff
@@ -51,15 +50,19 @@ Prerequisite: You have to **sign out** from all existing accounts.
...
+Prerequisite: You have to be **signed in**. This is important because we expect certain components to be refreshed automatically.
 
-Onboarding through GitLab is similar to [Sign in with GitHub](#sign-in-with-github), and the authorization is also done
-through the browser. Expected behaviour is identical.
+1. Navigate to `Settings` > `Sourcegraph & Cody`.
+2. Remove all accounts and apply settings.
+
+#### Expected behaviour
+
+* Cody toolbar is automatically refreshed and the user is greeted with the `Welcome to Cody` panel.
```
Comment (mkondratek) [inline TESTING.md:64]: nit: `s/toolbar/tool window/g`

Comment (exigow) [reply TESTING.md:64]: Sorry, but what do you mean by `s/toolbar/tool window/g`? :smile:

Comment (pkukielka) [reply TESTING.md:64]: It's sed replace syntax, it means you have typo or other text bug which reviewer is asking you to fix ;)
`g` means `global`, so it replaces all occurences.

Comment (pkukielka) [reply TESTING.md:64]: I took a liberty to merge it, we can improve it next time.

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**